astroquery/utils/system_tools.py

```diff
@@ -50,9 +25,16 @@ def gunzip(filename):
         Name of the decompressed file (or input filename if gzip is not
         available).
     """
-    # ".fz" denotes RICE rather than gzip compression
-    if __is_gzip_found and not filename.endswith('.fz'):
-        subprocess.call(_unzip_cmd(filename), stdout=DEVNULL, stderr=DEVNULL)
+    import shutil
+    import gzip
+
+    # @EP 09/01/20: system-wide 'gzip' was removed, Python gzip used instead.
```

Comment:
no need to date/sign comments. The git history/blame is more reliable to trace things back.